test/parallel/test-repl-external-module.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const { execSync } = require('child_process');
+
+console.log(process.execPath);
+
+const result = execSync(process.execPath, {
+  env: {
+    ...process.env,
+    NODE_EXTERNAL_REPL_MODULE: require.resolve(
+      '../fixtures/external-repl-module.js'),
```

Comment:
This will probably fail in CI somewhere because I'm pretty sure we have something where there's a weird NFS split or symlink or something for test/fixtures. I forget the details, it might involve environment variables too, but ANYWAY I think you'll want to use `const fixtures = require('../common/fixtures');` above and then change this:

```suggestion
     fixtures.path( 'external-repl-module.js'),
```

Reply:
Documentation for `fixtures` is at https://github.com/nodejs/node/tree/master/test/common#fixtures-module. I wouldn't mind getting rid of it at some point if it's unnecessary, but I think it does (or at least used to) paper over some CI funkiness.

Reply:
(That probably gets rid of the need for `require.resolve()` unless that's part of what is being tested itself?)